SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/DefaultTaskScheduler.h

```diff
@@ -235,7 +235,7 @@ namespace sofa  {
             
             unsigned m_workerThreadCount;
             
-            bool m_workerThreadsIdle;
+            volatile bool m_workerThreadsIdle;
```

Comment:
Can you explain why `volatile` is desirable here?

Reply:
If the task scheduler system works correctly there should be only one thread modifying this variable and multiple threads reading it. The volatile prevents the multiple threads to cache this variable.